### DISCUSSION TITLE
Streaming Multipule Displays instead of just one?

### DISCUSSION THREAD

**Discussion by thatsysadmin** [LizardByte/community#26] [Sunshine] [answered]
Has anyone figured out how to get sunshine to capture 2 or 3 displays in a multi display setup yet?

**Comment by ReenigneArcher** [comment, marked as answer]
I don't believe sunshine currently allows this, although in theory it should be possible. RDP and many other screen sharing protocols do allow for capturing all connected displays.

**Reply by thatsysadmin** [reply]
VNC is slow, real slow, on top of painful configuration. That's one thing Windows gets right; graphical session management. I was thinking that Sunshine could sort of be used as a remote solution.

Thanks for helping me with Fedora 35 support again though!

**Reply by ReenigneArcher** [reply]
Did you ever get audio working?

Also, what all the steps required to use the rpm package? Going through the process of trying to better document Sunshine now.

Just install the rpm using `sudo rpm -i sunshine_fedora_35.rpm` ?

**Reply by thatsysadmin** [reply]
Not yet. It's weird because I've been launching sunshine as root and it still thinks it's being denied access.

On installation, you do `sudo dnf install ./sunshine_fedora_35.rpm` and then you can launch it with `sudo sunshine /etc/sunshine/sunshine.conf`
I'm still trying to figure out the permissions problems with it not being able to launch in a user context.
